ArduCopter/Copter.cpp

```diff
@@ -26,7 +26,7 @@ const AP_HAL::HAL& hal = AP_HAL::get_HAL();
 Copter::Copter(void)
     : DataFlash(g.log_bitmask),
     flight_modes(&g.flight_mode1),
-    control_mode(STABILIZE),
+    control_mode(INITIALISING),
```

Comment:
You are now initializing it in class declaration, so no need to have this here.

There is a problem below though, `flightmode` is set to stabilize.